### PR TITLE
add v3 staking panel style

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

Added styles for the staking panel on the assets page

Screenshot 2024-01-14 at 12 57 45 PM

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**Remaining Todos:**
- [ ] Add a tooltip to the remaining Days (@ayumitk will add it in another PR)

**Need logics:**
- [x] Period
- [x] Remaining days
- [x] Available to withdraw amount
- [x] Re-lock amount

Screenshot 2024-01-14 at 1 16 12 PM

Screenshot 2024-01-14 at 1 15 36 PM

### CODE REVIEW

**Comment by Kahonnohak** [comment]
All looks good to me!

One small thing on mobile to adjust if you have time
1) Available to Vote -> 2 lines maybe?
2) Unbonding amount -> bottom fix
3) Remaining days -> Centre fix

And sorry my mistake
4) no days needed for the bonus (My Staking)
5) Are you adding this in this PR?


@bobo-k2 
Could you please help adding logics for 

My Staking
1) Add period
2) Add eras on the days (its technically different but please let us use days in the UI)

My dApps
 3) Would it be possible to separate the staking per projects also if the amount has bonus or not?
 eg. 
 dApp A  xxxx ASTR  Bonus
 dApp A  xxxx ASTR      -    

Unbonding
4) Available to withdraw
5) Re-lock + staking modal

**Comment by bobo-k2** [comment]
> My dApps
> 3) Would it be possible to separate the staking per projects also if the amount has bonus or not?
eg.
> dApp A xxxx ASTR Bonus
> dApp A xxxx ASTR -

I am sorry, I don't get the above

**Comment by ayumitk** [comment]
@Kahonnohak I've fixed 1,2,3,4.

> 5) Are you adding this in this PR?

sorry, I don't get this. What does this mean?

**Comment by ayumitk** [comment]
> At certain browser widths (˜1300px, ˜1050px, ˜700px), arrows arrangement looks weird

@bobo-k2 I've fixed this. And thank you for adding logics🙏

**Comment by Kahonnohak** [comment]
> @Kahonnohak I've fixed 1,2,3,4.
> 
> > 5. Are you adding this in this PR?
> 
> sorry, I don't get this. What does this mean?

oops sorry did not notice the message, its the tooltip. It is okay not to include in this PR.

Screenshot 2024-01-15 at 16 51 26

**Comment by Kahonnohak** [comment]
@ayumitk thank you for the fix! I am so sorry, i missed a few places. 

1) Dark theme has not been configured
Screenshot 2024-01-16 at 16 02 36


2) No relation to this PR but the banner is covering the account area
Screenshot 2024-01-16 at 16 02 06


3) some spacing - sorry
 A) Just realise why I did not do this, it is much better to have all balances at the right side, so please change the position of Remaining era <-> amount, also Bonus <-> amount on Unbonding and my dApp section
 
 B) On mobile, the SBY box SBY has a big space on the right but this is because of ASTR position? Would like all amount lined and but ticker has 4 or 3 letter.
 
 C) please make here 32px same as other box
 
 4) Tab underline is too long on mobile, if you could easily adjust please do
Screenshot 2024-01-16 at 16 50 43
Screenshot 2024-01-16 at 16 02 17

**Comment by ayumitk** [comment]
> A) Just realise why I did not do this, it is much better to have all balances at the right side, so please change the position of Remaining era <-> amount, also Bonus <-> amount on Unbonding and my dApp section

@Kahonnohak If I switch the Remaining era <-> amount position, how about the Available to withdraw and Re-lock section? It would be like this: 

Screenshot 2024-01-16 at 1 11 39 PM

**Comment by ayumitk** [comment]
> B) On mobile, the SBY box SBY has a big space on the right but this is because of ASTR position? Would like all amount lined and but ticker has 4 or 3 letter.

@Kahonnohak Yes, it's because of ASTR. Also, the amounts and symbols are vertically lined. Here is the current layout.

Screenshot 2024-01-16 at 1 32 03 PM

We can make amounts and symbols align right if you are okay with not vertically lined like this.

Screenshot 2024-01-16 at 1 42 34 PM

**Comment by Kahonnohak** [comment]
> > A) Just realise why I did not do this, it is much better to have all balances at the right side, so please change the position of Remaining era <-> amount, also Bonus <-> amount on Unbonding and my dApp section
> 
> @Kahonnohak If I switch the Remaining era <-> amount position, how about the Available to withdraw and Re-lock section? It would be like this:
> 
> Screenshot 2024-01-16 at 1 11 39 PM

My bad, of course. Sorry no change needed for this. Thank you for checking.

**Comment by Kahonnohak** [comment]
Thank you! lets double check the details on Astar mainnet!

**Comment by Kahonnohak** [comment]
Two problems on Asset page (another PR?)
- On this PR I see the rewards box next to the account but on the other PR I don't see it.
- Lets change the order 
Account
SBY
Staking
(project)
Assets

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Comment by ayumitk** [comment]
> - On this PR I see the rewards box next to the account but on the other PR I don't see it.
> - Lets change the order

@Kahonnohak I've removed the rewards box and changed the order

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -742,8 +742,8 @@ export default {
     voteLockedTokensWarning:
       'Be sure you vote on dapps otherwise those tokens are not eligible for any rewards.',
     rewardsWillBeClaimed: 'Your unclaimed rewards will be claimed.',
-    basicRewards: 'Basic rewards',
-    bonusRewards: 'Bonus rewards',
+    basicRewards: 'Available to claim',
```
Comment (impelcrypto) [inline src/i18n/en-US/index.ts:745]:
```suggestion
    availableToClaim: 'Available to claim',
```

File: src/i18n/en-US/index.ts [hunk 2]
```diff
@@ -762,13 +762,21 @@ export default {
     myStaking: 'My Staking',
     myDapps: 'My dApps',
     unbonding: 'Unbonding',
-    totalEstimatedRewards: 'Total estimated rewards',
+    totalEstimatedRewards: 'Claim estimated Rewards',
```
Comment (impelcrypto) [inline src/i18n/en-US/index.ts:765]:
```suggestion
    claimEstimatedRewards: 'Claim estimated Rewards',
```

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**